examples/standard/http-redirect-rewrite/httproute-redirect-http.yaml

```
@@ -0,0 +1,18 @@
+#$ Used in:
+#$ - site-src/api-types/httproute.md
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: http-filter-redirect
+spec:
+  parentRefs:
+  - name: redirect-gateway
+    sectionName: http
+  hostnames:
+  - redirect.example
+  rules:
+    - filters:
+        - type: RequestRedirect
+          requestRedirect:
+            scheme: https
+            statusCode: 301
```

Nit: Recommend updating indentation to match what you've done elsewhere.
```suggestion
  - filters:
    - type: RequestRedirect
      requestRedirect:
        scheme: https
        statusCode: 301
```

---

Updated!